jsons/Policies.json

```diff
@@ -182,7 +182,7 @@
 			},
 			{
 				"name": "Theocracy",
-				"uniques": ["+[33]% [Gold] from every [Grand Temple]","[+2 Gold] from every [Temple]","[+3 Gold] from every [Holy site]"],
+				"uniques": ["+[33]% Gold from every [Grand Temple]","[+2 Gold] from every [Temple]","[+3 Gold] from every [Holy site]"],
```

Comment:
iirc this should be [+33]% [Gold] from every [Grand Temple]

Reply:
Then what changed, nothing? Soulinor said that he fixed mod deprecations. Are you sure?

Reply:
The + sign should be moved instead the []

Reply:
This is the exact unique you're using:
https://github.com/yairm210/Unciv/blob/master/docs/uniques.md#amount-stat-from-every-tilefilterspecialistbuildingname

Reply:
resolved in https://github.com/ravignir/RekMOD/pull/43/commits/d98b1105a0e0d4f0661c3954f3b1811a2899ab7c